sdk/evaluation/azure-ai-evaluation/setup.py

```diff
@@ -71,8 +71,11 @@
         "azure-identity>=1.16.0",
         "azure-core>=1.30.2",
         "nltk>=3.9.1",
-        "azure-storage-blob>=12.10.0"
+        "azure-storage-blob>=12.10.0",
     ],
+    extras_require={
+        "pyrit": ["pyrit @ git+https://github.com/Azure/PyRIT.git"]
```

Comment:
you're going to want to pin that to a specific version because PyRIT introduces breaking changes between minor versions.